common/buildcraft/builders/tile/TileQuarry.java

```diff
@@ -291,6 +296,20 @@ public boolean hasCapability(Capability<?> capability, EnumFacing facing) {
         return super.getCapability(capability, facing);
     }
 
+
+
+    @Override
+    @SideOnly(Side.CLIENT)
+    public AxisAlignedBB getRenderBoundingBox() {
+        return box == null ? super.getRenderBoundingBox() : BoundingBoxUtil.makeFrom(getPos(), box);
```

Comment:
BoundingBoxUtil checks to see if box is null, and if it is it just uses "pos". So you don't need the null check, and can replace this with
````java
public AxisAlignedBB getRenderBoundingBox() {
    return BoundingBoxUtil.makeFrom(getPos(), box);
}
````
and it won't ever throw an NPE.